### PR TITLE
Moved output_files page to same level as input_files (was a subdir).

### PR DESCRIPTION
**Pull request by krefson**
Enhanced content of output_files to describe most known CASTEP output files.

### CODE REVIEW

File: docs/documentation/Output_Files/output_files.md [hunk 1]
```diff
@@ -45,6 +56,12 @@ ASCII. same data as `.chdiff` in human readable format. Only written if `calcula
 * `.xrd_sf`
 ASCII. X-ray structure factors. See the [documentation page](documentation/XRD/overview)
```
Comment (oerc0122) [inline docs/documentation/Output_Files/output_files.md]:
```suggestion
ASCII. X-ray structure factors. See the [documentation page](/documentation/XRD/overview)
```

File: docs/documentation/Output_Files/output_files.md [hunk 2]
```diff
@@ -45,6 +56,12 @@ ASCII. same data as `.chdiff` in human readable format. Only written if `calcula
...
 
+*  `.elf`
+Binary. Result of ELF calculation.
+
+* `.elf_fmt`
+ASCII.   Result of ELF calculation.Only written if `write_formatted_elf : T `.
```
Comment (oerc0122) [inline docs/documentation/Output_Files/output_files.md]:
```suggestion
ASCII. Result of ELF calculation. Only written if `write_formatted_elf : T `.
```

File: docs/documentation/Output_Files/output_files.md [hunk 3]
```diff
@@ -55,6 +72,9 @@ ASCII. State of the system (coordinates, unit cell etc) at each step of the geom
 * `*.md`
 ASCII. State of the system (coordinates, unit cell etc) at each step of the molecular dynamics simulation. Same format as the `.geom` file.  See for specification. Can be used to animate the geometry optimisation - can be
```
Comment (oerc0122) [inline docs/documentation/Output_Files/output_files.md]: Incomplete sentence.

**Review by oerc0122 — CHANGES_REQUESTED** [review]
This needs updating in the `mkdocs.yml` too, to point to new folder.

File: docs/documentation/Output_Files/output_files.md (hunk 2 above)
Comment (krefson) [reply docs/documentation/Output_Files/output_files.md]: Fixed

File: docs/documentation/Output_Files/output_files.md (hunk 3 above)
Comment (krefson) [reply docs/documentation/Output_Files/output_files.md]: Fixed

File: docs/documentation/Output_Files/output_files.md (hunk 1 above)
Comment (krefson) [reply docs/documentation/Output_Files/output_files.md]: Fixed

**Review by jryates — APPROVED** [review]

**Pull request merged**